governance/reviews/cloudevents-08-29-2023.md

```diff
@@ -0,0 +1,210 @@
+
+Governance files are not generally interlinked, nor do they seem to refer to each other at all; it's up to the individual contributor to figure out which files apply to their role.  This could be improved a great deal as part of a general governance cleanup.
+
+Many bits of governance documentation are found in different files than one would conventionally expect them to be found.  Contributor information is found in Governance or Maintainers files; decision making requirements in Contributors files.
+
+### Documentation Content
+
+<!--- Provide the commit of the file under evaluation as a point-in-time reference to this review. --->
+
+The following table details the governance areas expected for a project. Coverage is indicated by Complete, Partial, and Missing.
+* Complete - the content of the governance documentation is fully detailed and does not leave any question to the reader.
+* Partial - the content of the governance documentation is missing some information and would leave the reader with questions or some level of misunderstanding.
+* Missing - the documentation is absent, wholly undiscoverable, or woefully inadequate in meeting the objectives of that governance content. The reader cannot act on the content that is available.
+* Unknown - status cannot be assessed at this time
+
+| Governance Area | Coverage | Documents | Finding Notes |
+|:----------------|:--------:|:------:|:--------------|
+| Project Purpose | Complete | [README](https://github.com/cloudevents/spec) | |
+| Maintainer List | Partial | [OWNERS](https://github.com/cloudevents/spec/blob/main/OWNERS), [Spreadsheet](https://docs.google.com/spreadsheets/d/1bw5s9sC2ggYyAiGJHEk7xm-q2KG6jyrfBy69ifkdmt0/edit?pli=1#gid=0) | See discussion above |
+| Code of Conduct | Complete | Uses link to CNCF CoC | |
+| Contributor Guide | Partial | [CONTRIBUTING](https://github.com/cloudevents/spec/blob/main/docs/CONTRIBUTING.md) | See discussion above |
+| Contributor Ladder | Partial | [GOVERNANCE](https://github.com/cloudevents/spec/blob/main/docs/GOVERNANCE.md) | Several roles exist, but qualifications and advancement are not explained |
+| Maintainer Lifecycle | Missing | | How to become, or retire, a Voting Member is not explained anywhere |
+| Decision-making | Complete | [GOVERNANCE](https://github.com/cloudevents/spec/blob/main/docs/GOVERNANCE.md), [SDK Governance](https://github.com/cloudevents/spec/blob/main/docs/SDK-GOVERNANCE.md) | Nonlinear, but explained in exacting detail |
+| Code and Docs Ownership | Complete | [OWNERS](https://github.com/cloudevents/spec/blob/main/OWNERS), plus each repo | |
+| Security Reporting and response | Missing | | A security team exists but is not documented |
+| Communication and Meetings | Partial | [GOVERNANCE](https://github.com/cloudevents/spec/blob/main/docs/GOVERNANCE.md) | meeting process, no calendar.  should link to SDK comms info |
```

Comment:
The main README points to the CNCF calendar: https://www.cncf.io/calendar/ and we're listed on there.

Added some text to the main README telling people to check the README of each SDK for its comm stuff. I'd prefer not to duplicate it in the main repo to avoid it getting out of sync or requiring multiple PRs when things change.

Reply:
Some of the SDKs have meeting/comms info, but some don't.

Reply:
I just checked them all and they all seem to have a section like this: https://github.com/cloudevents/sdk-javascript#community  which one do you think I missed?

Reply:
I think you've filled in the missing ones, I need to do a re-check and then I can update this.